factfinder/data/metadata.json

```diff
@@ -111,8 +111,8 @@
         "source": ""
     },
     {
-        "pff_variable": "pop2010",
```

Comment:
can we do `decennial_pop`,  let's just prefix decennial variables with `decennial` so that we can bulk replace them later on